test/com/google/enterprise/adaptor/database/UniqueKeyTest.java

```diff
@@ -450,9 +443,9 @@ private void testUniqueElementsRoundTrip(String elem1, String elem2)
       ResultSet rs = executeQueryAndNext("select * from data");
       String id = uk.makeUniqueId(rs);
 
-      String sql = "select * from data where a = ? and b = ?";
-      try (Connection connection = getConnection();
-           PreparedStatement ps = connection.prepareStatement(sql)) {
+      try {
```

Comment:
I think I wouldn't have changed this one, since it wasn't broken and is doing some things manually anyway.